tutugodfrey/maintenance-tracker
# #161199304 fix failing tests

#### What does this PR do?
- Fixes failing tests

#### Description of Task to be completed?
- Modify request details and expected values where necessary

#### How should this be manually tested?
- Clone the repo and check out the branch. Run `npm install` to install dependencies.  Set up your environment using the sample in .example.env file. Run `npm test` and confirm that all tests pass

#### Any background context you want to provide?
- Failing tests are due to a modification in the return value when a request is created. Initially, only the created request is returned. Due to changes in requirement, the returned value has to contain an object with properties `request` and `user` where the user is the associated admin for the request be made. 

#### What are the relevant pivotal tracker stories?
#161199304
